assets/sass/main.scss

```diff
@@ -41,6 +41,8 @@ $font-size-base:  $aplus-font-size-base;
 /*!
  * A+ Styles
  */
+
+@import 'components/topbar';
```

Comment:
Is it necessary to import topbar first or could we maintain alphabetic sorting in these imports?

Reply:
based on the current implementation we can keep using alphabetic sorting. I will move this line below.